server/settings.py

```diff
@@ -186,6 +186,7 @@ def env(variable, fallback_value=None):
 MAIL_USERNAME = env('MAIL_USERNAME', 'liveblogsf@gmail.com')
 MAIL_PASSWORD = env('MAIL_PASSWORD', 'fabric2010')
 ADMINS = [MAIL_USERNAME]
+MAIL_FROM = env('MAIL_FROM', 'liveblogsf@gmail.com')
```

Comment:
since MAIL_FROM field should be supported bu superdesk-core `email` module, MAIL_FROM env var should be read by `factory.default_settings`, not here